tawazi/__init__.py

```diff
@@ -1,3 +1,17 @@
+# !/bin/python3
+# isort: skip_file
+
+# define import order
+# from . import config as config
+# from . import consts as consts
+# from . import errors as errors
+# from . import node as node
+# from . import digraph as digraph
+# from . import dag as dag
+# from . import decorators as decorators
+
+
+# exposing useful objects / Classes
```

Comment:
why is everything commented like that ? Can we remove those lines ?

Reply:
Yes removing them